### PR TITLE
:triangular_ruler: Rename and fix APISpark extension

### PR DESCRIPTION
**Pull request by quilicicf**

### CODE REVIEW

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/package.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<HTML>
+<BODY>
+Integration with Restlet Cloud platform. Help document and manage your web API.
+
+@since Restlet 2.3
```
Comment (thboileau) [inline modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/package.html]: I wonder if we can precise that the extension has been renamed from apispark to platform.

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java [hunk 2]
```diff
@@ -214,7 +208,7 @@ public static void process(String[] args) throws TranslationException {
         }
         if (createNewVersion) {
             if (createNewCell || updateCell) {
-                failWithErrorMessage("In create new version mode, you can't use -d (--create-descriptor), -c (--create-connector) or -n (--new-version). ");
+                failWithErrorMessage("In create new version mode, you can't use -c (--create-connector) or -n (--new-version). ");
```
Comment (thboileau) [inline modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: :+1:

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java [hunk 3]
```diff
@@ -225,64 +219,52 @@ public static void process(String[] args) throws TranslationException {
         }
         if (updateCell) {
             if (createNewCell || createNewVersion) {
-                failWithErrorMessage("In update mode, you can't use -d (--create-descriptor), -c (--create-connector) or -N (--new-version). ");
+                failWithErrorMessage("In update mode, you can't use -c (--create-connector) or -N (--new-version). ");
```
Comment (thboileau) [inline modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: I guess there is a typo on the "-N" attribute. it should be "-n", right?

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java [hunk 4]
```diff
@@ -320,11 +296,10 @@ public boolean isLoggable(LogRecord record) {
...
         }
 
-        IntrospectionUtils.sendDefinition(definition, ulogin, upwd, serviceUrl,
-                cellType, cellId, cellVersion, createNewCell, createNewVersion,
-                updateCell, updateStrategy, LOGGER);
+        IntrospectionUtils.sendDefinition(definition, ulogin, upwd, serviceUrl, cellId, cellVersion, createNewCell,
+                createNewVersion, updateCell, updateStrategy, LOGGER);
 
-        LOGGER.info("Instrospection complete");
+        LOGGER.info("Introspection complete");
```
Comment (thboileau) [inline modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: :+1:

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java [hunk 5]
```diff
@@ -334,36 +309,31 @@ private static void printHelp() {
...
         cli.print();
         cli.print1("[actions]");
-        cli.print12("-d, --create-descriptor",
-                "Creates a new descriptor from introspection.");
-        cli.print12("-c, --create-connector",
-                "Creates a new connector from introspection.");
+        cli.print12("-c, --create-connector", "Creates a new connector from introspection.");
         cli.print12(
                 "-n, --new-version",
                 "Creates a new version of the descriptor/connector identified by the -i (--id) option");
```
Comment (thboileau) [inline modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: `Creates a new version of the connector identified by the`

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java [hunk 6]
```diff
@@ -384,8 +354,8 @@ private static void printHelp() {
                 "-s, --update-strategy strategy",
                 "Specifies the update strategy.",
                 "Available strategies:",
-                "- update: (default) new objects will be added to the APISpark's descriptor/connector, primitive fields of existing objects will be updated. Nothing will be deleted.",
-                "- replace: deletes all the information in the descriptor/connector on APISpark's and fills it again with introspected definition.");
+                "- update: (default) new objects will be added to the Restlet Cloud's descriptor/connector, primitive fields of existing objects will be updated. Nothing will be deleted.",
```
Comment (thboileau) [inline modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: `new objects will be added to the Restlet Cloud's connector`

**Comment by thboileau** [comment]
Hi @quilicicf , few comments.

Could you add an entry in the "build/tmpl/text/changes.txt file"

```
    - Misc
      - Renamed apispark extension to platform, and removed the support of API description.
```

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/package.html (hunk 1 above)
Comment (quilicicf) [reply modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/package.html]: Why not. I have absolutely no idea where this is shown in the end though ?

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java (hunk 3 above)
Comment (quilicicf) [reply modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: Bull's eye!

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java (hunk 5 above)
Comment (quilicicf) [reply modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: Bull's eye!

File: modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java (hunk 6 above)
Comment (quilicicf) [reply modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java]: Bull's eye!

**Pull request merged**